components/dashboard/src/components/PrebuildLogs.tsx

```diff
@@ -81,9 +81,8 @@ export default function PrebuildLogs(props: PrebuildLogsProps) {
 
     useEffect(() => {
         switch (workspaceInstance?.status.phase) {
-            // Preparing means that we haven't actually started the workspace instance just yet, but rather
-            // are still preparing for launch. This means we're building the Docker image for the workspace.
-            case "preparing":
```

Comment:
addition

Reply:
treat both "preparing" and "building" as equivalent for this PR.

Reply:
89c1cb1ea18b1174a9ca07aa2713415b60dd1cbc